pkg/apis/minio.min.io/v2/types.go

```diff
@@ -128,7 +128,7 @@ type TenantSpec struct {
 	//
 	// If provided, the MinIO Operator adds the specified environment variables when deploying the Tenant resource.
 	// +optional
-	Env []corev1.EnvVar `json:"env,omitempty"`
+	Env []EnvVar `json:"env,omitempty"`
```

Comment:
This is fine, for the sake of backward compatibility, but we should modify MinIOJob to read the same, specially since MinIOJob is on a young enough stage to do it safely (alpha)

Reply:
Created https://github.com/minio/operator/issues/2302.